src/uikit/Card/index.js

```diff
@@ -12,15 +13,27 @@ const Card = ({
   scrollable,
   title,
   inactive = false,
+  showHeader = true,
 }) => {
   const DefaultHeader = <CardHeader title={title} />;
 
   return (
     <CardWrapper className={className} inactive={inactive}>
-      <HeaderWrapper inactive={inactive}>{Header || DefaultHeader}</HeaderWrapper>
+      {showHeader && <HeaderWrapper inactive={inactive}>{Header || DefaultHeader}</HeaderWrapper>}
       <Content scrollable={scrollable}>{children}</Content>
     </CardWrapper>
   );
 };
 
+Card.propTypes = {
+  Header: PropTypes.node,
+  Content: PropTypes.func,
+  children: PropTypes.node,
+  className: PropTypes.string,
+  scrollable: PropTypes.bool,
+  title: PropTypes.node,
+  inactive: PropTypes.bool,
+  showHeader: PropTypes.bool,
+};
+
```

Comment:
@jberube I hope this gives you a smile :)

Reply:
Yes! :smile_cat: 

None of the props are marked as `.isRequired`, are thay all optional?

What happen if you do that?
```
<Card />
```

Marking the the required props as so and providing defaults otherwise, when necessary, would make it a 10 to me.

Reply:
```
<Card />
```
would work just fine, would just get an empty card :P
![screen shot 2019-02-15 at 10 01 19 am](https://user-images.githubusercontent.com/11821106/52864708-bbb1c580-3108-11e9-8852-5e7e151261dd.png)

Reply:
:+1: